lib/index.js

```diff
@@ -43,10 +45,15 @@ function VM (opts = {}) {
   this._precompiled['0000000000000000000000000000000000000002'] = num02
   this._precompiled['0000000000000000000000000000000000000003'] = num03
   this._precompiled['0000000000000000000000000000000000000004'] = num04
+  this._precompiled['0000000000000000000000000000000000000006'] = num06
+  this._precompiled['0000000000000000000000000000000000000007'] = num07
 
   if (this.opts.activatePrecompiles) {
-    for (var i = 1; i <= 4; i++) {
-      this.trie.put(new Buffer('000000000000000000000000000000000000000' + i, 'hex'), new Account().serialize())
+    for (var i = 1; i <= 7; i++) {
+      // temporarily skip 5 till EIP 198 added
```

Comment:
I think there's no harm setting it. This code only ensures those accounts are touched and thus the first call to them won't incur an account creation fee.